server/utils/codes.js

```diff
@@ -0,0 +1,8 @@
+exports.statusCodes = {
+  200: 200,
+  201: 201,
+  400: 400,
+  404: 404,
+  409: 409,
+  500: 500,
```

Comment:
would you mind using names instead of numbers as keys? I mean, `success: 200, ...`

Reply:
Noted.